reference/examples/LazyCreateAccount.md

```diff
@@ -0,0 +1,14 @@
+Lazy-create a new account using a public-address via the CryptoCreate transaction.
+
+Reference: HIP-583
+
+## Example 1
+- Create an Ethereum account address/public-address (example: 0xb794f5ea0ba39494ce839613fffba74279579268)
```

Comment:
Where does one create an Eth account? @Nana-EC

Reply:
Just a regular ECDSA key pair, this is already supported by the SDK